h/views/api/users.py

```diff
@@ -92,14 +91,11 @@ def update(context, request):
     This API endpoint allows authorised clients (those able to provide a valid
     Client ID and Client Secret) to update users in their authority.
     """
-    schema = UpdateUserAPISchema()
-    appstruct = schema.validate(_json_payload(request))
+    appstruct = UpdateUserAPISchema().validate(_json_payload(request))
 
-    user_update_service = request.find_service(name="user_update")
-    user = user_update_service.update(context.user, **appstruct)
+    user = request.find_service(name="user_update").update(context.user, **appstruct)
 
-    presenter = TrustedUserJSONPresenter(user)
-    return presenter.asdict()
+    return TrustedUserJSONPresenter(user).asdict()
```

Comment:
This is all the same stuff, just on fewer lines so I could understand what was going on